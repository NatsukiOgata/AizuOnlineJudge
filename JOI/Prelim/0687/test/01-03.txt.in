100
61 41 25 87 82 48 16 31 81 13 5 3 75 74 32 50 72 14 96 44 59 63 20 52 58 73 93 23 67 21 45 78 53 1 68 46 7 22 85 6 64 94 40 98 100 39 30 34 47 91 15 29 12 19 10 84 65 33 80 57 62 37 9 42 26 18 43 77 92 24 35 56 28 95 11 49 88 70 86 27 71 83 55 76 79 2 60 8 51 66 36 90 69 97 99 17 4 89 54 38
