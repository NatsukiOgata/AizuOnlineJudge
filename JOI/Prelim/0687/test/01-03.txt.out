2186
2764
